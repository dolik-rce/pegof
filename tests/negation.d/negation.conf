input negation.d/negation.peg
optimize double-negation
