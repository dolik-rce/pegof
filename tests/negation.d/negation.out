main <-
    [a-z]
    / [^A-Z]
    / "\n"
    / (. . .)
